### DISCUSSION TITLE
Redirect Http to Https

### DISCUSSION THREAD

**Discussion by kodedewe** [brefphp/bref#1120] [Support]
Hi, guys any idea how to redirect HTTP to HTTPS?
Used .htaccess but it is not working.

I need this solution since clients on their browser can type http://domain.com rather than https://domain.com
Thanks

**Comment by t-richard** [comment]
Hi there!

`.htaccess` is an Apache-only configuration file.

When running your code in AWS Lambda, you don't run Apache and API Gateway is replacing the webserver so you need to find "an AWS way" of doing HTTP to HTTPS redirects.

The most straight forward way for you to do so is to follow this documentation: https://bref.sh/docs/websites.html

Thiw will put your API gateway behind Cloudfront and will come with redirects out of the box.

Feel free to reach out again if you need further guidance.
